backend/src/auth/controllers/logOutController.ts

```diff
@@ -0,0 +1,7 @@
+import { Request, Response } from "express";
+
+export function logoutController(req: Request, res: Response) {
+    req.session.user = null;
```

Comment:
I just think, maybe it's better to reset the whole session, not just an user?
otherwise you will keep all session data in the storage even if user already gone.